csrc/cpu/torch_bindings.cpp

```diff
@@ -84,6 +89,26 @@ TORCH_LIBRARY_EXPAND(TORCH_EXTENSION_NAME, ops) {
       "                 Tensor! key, int head_size,"
       "                 Tensor cos_sin_cache, bool is_neox) -> ()");
   ops.impl("rotary_embedding", torch::kCPU, &rotary_embedding);
+
+  // Quantization
+  // Compute int8 quantized tensor for given scaling factor.
+  ops.def(
+      "static_scaled_int8_quant(Tensor! out, Tensor input, Tensor scale) -> "
+      "()");
+  ops.impl("static_scaled_int8_quant", torch::kCPU, &static_scaled_int8_quant);
+  // Compute int8 quantized tensor and scaling factor
+  ops.def(
+      "dynamic_scaled_int8_quant(Tensor! out, Tensor input, Tensor! scale) -> "
+      "()");
+  ops.impl("dynamic_scaled_int8_quant", torch::kCPU,
+           &dynamic_scaled_int8_quant);
+  // W8A8 GEMM, supporting symmetric per-tensor or per-row/column
+  // quantization.
+  ops.def(
+      "cutlass_scaled_mm(Tensor! out, Tensor a,"
+      "                  Tensor b, Tensor a_scales,"
+      "                  Tensor b_scales, Tensor? bias) -> ()");
+  ops.impl("cutlass_scaled_mm", torch::kCPU, &int8_scaled_mm);
```

Comment:
I think it is confusing to call this cpu implementation `cutlass_scaled_mm`, especially since this only supports int8 while the GPU cutlass_scaled_mm supports fp8 and int8. I would prefer to keep int8 and possibly dnnl in the name.

I understand reusing this name likely provides some ease in re-using the testing we have for cutlass_scaled_mm, so I'm open to thoughts here. cc @tlrmchlsmth

Reply:
Using the same function signature make the kernel replacement seamlessly. The kernel for CPU will check the input types and throw error message when they are not INT8.

Maybe change ```cutlass_scaled_mm``` to a general name is more clear, like ```scaled_mm``` or ```compressed_tensor_scaled_mm```.

Reply:
>Maybe change cutlass_scaled_mm to a general name is more clear, like scaled_mm

I'm in favor of this as long as we maintain a consistent interface. (I don't think `compressed_tensor_scaled_mm` is right though, since we use the kernels for other model formats, like fb_gemm)